modules/LaneDetection/include/LaneDetection/LaneDetector.hpp

```diff
@@ -1,15 +1,63 @@
-#ifndef LANE_DETECTION_CLASSLANEDETECTION_HPP
-#define LANE_DETECTION_CLASSLANEDETECTION_HPP
+//
+// Created by nahye on 23. 11. 7.
+//
+
+#ifndef LANE_DETECTION_LANEDETECTOR_HPP
+#define LANE_DETECTION_LANEDETECTOR_HPP
```

Comment:
```suggestion
#ifndef LANE_DETECTION__LANEDETECTOR_HPP
#define LANE_DETECTION__LANEDETECTOR_HPP
```
폴더 구조는 구분자 하나 더 넣는 것으로 하죠!